cucumber.js

```diff
@@ -5,7 +5,6 @@ module.exports = {
     paths: ['./e2e/features/**/*.feature'],
     import: ['./e2e/hooks/hooks.mjs', './e2e/steps/**/*.mjs', './e2e/steps/**/*.js'],
     dryRun: false,
-    publishQuite: true,
```

Comment:
I think we need this one. Think CI could pass even if it shouldnt pass if we remove it

Reply:
There was a typo there and `publishQuiet` is no longer a valid config option based on https://github.com/cucumber/cucumber-js/blob/main/docs/configuration.md though `--publish-quiet` might work on the command line